patterns/mixed/variables.tf

```diff
@@ -639,4 +639,10 @@ variable "override" {
   default     = false
 }
 
+variable "override_json_string" {
```

Comment:
We probably want to have an example using the override_json_string input, and associated tests eventually. I don't think it's blocking this PR tough.